src/Compilers/Core/Portable/NativePdbWriter/PdbWriter.cs

```diff
@@ -778,14 +778,21 @@ private static object CreateSymWriterWorker()
             {
                 try
                 {
-                    var guid = new Guid(SymWriterClsid);
-                    if (IntPtr.Size == 4)
+                    try
                     {
-                        CreateSymWriter32(ref guid, out symWriter);
+                        var guid = new Guid(SymWriterClsid);
+                        if (IntPtr.Size == 4)
+                        {
+                            CreateSymWriter32(ref guid, out symWriter);
+                        }
+                        else
+                        {
+                            CreateSymWriter64(ref guid, out symWriter);
+                        }
                     }
-                    else
+                    catch (DllNotFoundException)
```

Comment:
>catch (DllNotFoundException) [](start = 20, length = 28)

Is this accurate enough to identify the specific failure we are trying to mitigate?

Reply:
Do you have any suggestions for making it more accurate?